Specs/DateTime/Hindi/DatePeriodExtractor.json

```diff
@@ -767,18 +767,6 @@
       }
     ]
   },
-  {
-    "Input": "मैं अक्टू. 2001 मिस कर रहा था",
```

Comment:
Can this case without the "," never happen?

Reply:
added test case with month abbreviation without '.'